src/main/java/edu/hillel/homework/lesson21/dao/DataBaseConnection.java

```diff
@@ -0,0 +1,34 @@
+package edu.hillel.homework.lesson21.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnection {
+
+    private static final String URL = "jdbc:mysql://localhost:3306/hillel";
+    private static final String USERNAME = "hillel";
+    private static final String PASSWORD = "hillel";
+
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        return DriverManager.getConnection(URL, USERNAME, PASSWORD);
+    }
+
+    public static Connection getConnection(String db_url, String db_username, String db_password) throws SQLException {
+        return DriverManager.getConnection(db_url, db_username, db_password);
+    }
+
+    public static void close(Connection connection) throws SQLException {
```

Comment:
Please, implement AutoClosable interface by this class and override close method to close connection properly

Reply:
Well done.